README.md

```diff
@@ -16,7 +16,7 @@ For further developer and user documentation please visit [the wiki](https://git
 ##### Minimum PHP Version
 * PHP >= 5.4
 
-##### Minimum ownCloud Version
+##### Minimum Nextcloud Version
 * >= 8.1
```

Comment:
I'm not completely sure. But wasn't 9.0 the first version of Nextcloud?
We should replace this version number accordingly.